lib/FrontendTool/FrontendTool.cpp

```diff
@@ -186,6 +186,22 @@ static bool printAsObjCIfNeeded(StringRef outputPath, ModuleDecl *M,
   });
 }
 
+/// Prints the C++  "generated header" interface for \p M to \p
+/// outputPath.
+///
+/// ...unless \p outputPath is empty, in which case it does nothing.
+///
+/// \returns true if there were any errors
+///
+/// \see swift::printAsCxx
```

Comment:
Nit: that function is spelled `swift::printAsCXX` with capital `XX` :)

Reply:
https://github.com/apple/swift/pull/40934